pyrelational/data/data_manager.py

```diff
@@ -352,3 +345,9 @@ def __str__(self) -> str:
         str_out += "Percentage Labelled: {}".format(str_percentage_labelled)
 
         return str_out
+
+
+def _check_dataset_has_len(dataset: Dataset) -> SizedDataset:
```

Comment:
can you make this a static method of the class? maybe rename _check_is_sized as well

Reply:
done